DXRModules/DeusEx/Classes/DXRMemes.uc

```diff
@@ -229,6 +230,34 @@ function AnyEntry()
                 v.Z = 70;
                 elec.move(v);
             }
+
+
+//Add me back in once we have "Randomizer" name textures!  Just change the textures below
+            foreach AllActors(class'#var(prefix)DXText',text)
+            {
+                text.bHidden=True; //Hide all the original text
```

Comment:
I wonder if we could make this work along with the old LDDP logo? or whatever might be there already